kubectl/kubectl_test.go

```diff
@@ -0,0 +1,12 @@
+package kubectl
+
+type TestRunner struct {
+	Runner
+
+	out []byte
+	err error
+}
+
+func (r TestRunner) Get(stdin []byte, ns string) ([]byte, error) {
```

Comment:
I'm confused why we have a `TestRunner` and a `PrintRunner` type. Aren't they both the same, but one just displays whatever you want it to and one actually displays the `kubectl` output? What's the idea behind `TestRunner`?